javascriptlib/src/mill/javascriptlib/TypeScriptModule.scala

```diff
@@ -418,34 +452,52 @@ trait TypeScriptModule extends Module { outer =>
    * removes need for node_modules prefix in import statements `node_modules/<some-package>`
    * import * as somepackage from "<some-package>"
    */
-  private[javascriptlib] def symLink: Task[Unit] = Task.Anon {
+  def symLink: Task[Unit] = Task.Anon {
```

Comment:
Let's name this a bit more verbosely, e.g. `createNodeModulesSymlink` or something. Currently it's far too cryptic